src/components/CurrentIngredient.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+class CurrentIngredient extends React.Component {
+  constructor(props) {
+    super(props);
+    this.deleteIngredient = this.deleteIngredient.bind(this);
+  }
+
+  deleteIngredient() {
+    this.props.onDeleteIngredient(this.props.index);
+  }
+
+  render() {
+    return (
+     <li key={this.props.index}>
+     <button id='delete-ingredient' onClick={this.deleteIngredient}>x </button>
```

Comment:
Space after the x. Guessing this will disappear anyway when the styling happens.

Reply:
Resolevd